README.md

```diff
@@ -46,6 +46,8 @@ SDK support with custom configuration for kafka stream, that is :
 * DialTimeout : Timeout duration during connecting to kafka broker (time.Duration)
 * ReadTimeout : Timeout duration during consume topic from kafka broker (time.Duration)
 * WriteTimeout : Timeout duration during publish event to kafka broker (time.Duration) 
+* LogMode : If it set true, eventstream will print log with debug level (event stream level) 
+* StrictValidation : If it set true, eventstream enable strict validation for event fields (boolean) 
```

Comment:
we need to document the default values for these settings

Reply:
sure, will update it
thanks